lang_tests/inter_func_call.ll

```diff
@@ -0,0 +1,71 @@
+; ModuleID = 'inter_func_call.c'
```

Comment:
We should stop pushing and then later removing these files. They don't really add anything to the PR.

Reply:
Yes, sorry, I just forget to remove them.

Reply:
You might want to only add files you want to push in the first place, then you don't need to remove them. Just do `git add filename` for each file you want to add or use a graphical tool like `gitk`.